src/libraries/Common/src/Interop/Windows/Advapi32/Interop.LsaSetForestTrustInformation.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Advapi32
+    {
+        [LibraryImport(Libraries.Advapi32)]
+        internal static partial uint LsaSetForestTrustInformation(SafeLsaPolicyHandle handle, in UNICODE_STRING target, IntPtr forestTrustInfo, int checkOnly, out IntPtr collisionInfo);
```

Comment:
```suggestion
        internal static partial uint LsaSetForestTrustInformation(SafeLsaPolicyHandle handle, in UNICODE_STRING target, IntPtr forestTrustInfo, [MarshalAs(UnmanagedType.U1)] bool checkOnly, out IntPtr collisionInfo);
```
The `BOOLEAN` data type is a single byte - https://learn.microsoft.com/windows/win32/api/ntsecapi/nf-ntsecapi-lsasetforesttrustinformation

Reply:
fwiw the handy table lists this: https://learn.microsoft.com/en-us/dotnet/standard/native-interop/best-practices#common-windows-data-types

Reply:
23ad205c7bd327a74318757457f3f0babed5e3eb